common/util.go

```diff
@@ -654,3 +658,20 @@ func FormatList(list []string, conjunction string) string {
 func BoolToPointer(b bool) *bool {
 	return &b
 }
+
+// Sanitize text to match confusables and remove diacritics
+var transformer = transform.Chain(norm.NFD, transform.RemoveFunc(isMn), norm.NFC)
+
+func isMn(r rune) bool {
+	return unicode.Is(unicode.Mn, r)
+}
+
+func SanitizeText(content string) string {
```

Comment:
Add some documentation above the function on what it does. "SanitizeText" could mean a lot of things.

Reply:
Where would I put the documentation? I'm kinda dumb